pilbox/app.py

```diff
@@ -138,7 +138,7 @@ def __init__(self, **kwargs):
         tornado.web.Application.__init__(self, self.get_handlers(), **settings)
 
     def get_handlers(self):
-        return [(r"/", ImageHandler)]
+        return [(r"/(.*)", ImageHandler)]
```

Comment:
I don't believe this is going to get merged upstream. Also, I'm not sure exactly what you're asking here?